src/Helper/MyParcelCollection.php

```diff
@@ -568,6 +569,90 @@ public function downloadPdfOfLabels($inline_download = false): void
         exit;
     }
 
+    /**
+     * Print labels directly to a printer
+     * 
+     * This method creates shipments (concepts) and sends them directly to a printer in one request.
+     * Based on the PDK implementation, it uses the same logic as createConsignments() but with
+     * a special Accept header that tells the API to print directly instead of returning a PDF.
+     *
+     * @param string $printerGroupId The ID of the printer group to print to
+     *
+     * @return array The API response, grouped by API key
+     * @throws AccountNotActiveException
+     * @throws ApiException
+     * @throws MissingFieldException
+     */
+    public function printDirect(string $printerGroupId): array
```

Comment:
Please adapt the existing `createConsignments` to be used with this additional ‘direct print’ header and endpoint, backwards compatible if possible (it is already used for return shipments with a boolean flag, that might need to change), to prevent much duplicate code.